WorkflowUI/Sources/Container/ContainerViewController.swift

```diff
@@ -116,7 +116,7 @@
 
         var wrapped: AnyWorkflow<Rendering, Output>
 
-        init<W: AnyWorkflowConvertible>(_ wrapped: W) where W.Output == Output, W.Rendering == Rendering {
+        init<W: AnyWorkflowConvertible>(_ wrapped: W) where W.Rendering == Rendering, W.Output == Output {
```

Comment:
This doesn't make a difference, does it?

Reply:
Nope. Just trying to be consistent.